alacritty_terminal/src/config/bell.rs

```diff
@@ -0,0 +1,105 @@
+use std::time::Duration;
+
+use log::error;
+use serde::{Deserialize, Deserializer};
+use serde_yaml::Value;
+
+use crate::config::{failure_default, Program, LOG_TARGET_CONFIG};
+use crate::term::color::Rgb;
+
+const DEFAULT_BELL_COLOR: Rgb = Rgb { r: 255, g: 255, b: 255 };
+
+#[serde(default)]
+#[derive(Deserialize, Clone, Debug, PartialEq, Eq)]
+pub struct BellConfig {
+    /// Visual bell animation function.
+    #[serde(deserialize_with = "failure_default")]
+    pub animation: BellAnimation,
+
+    /// Visual bell duration in milliseconds.
+    #[serde(deserialize_with = "failure_default")]
+    duration: u16,
+
+    // FIXME RGB deserialization always successedes and uses #000000 color. However it should
+    // fallback to a proper color. So it should be reworked either with
+    // https://github.com/alacritty/alacritty/pull/3507 or after that PR will land.
```

Comment:
I mean that you'll start doing manual deserialization, right? So it'll be possible to do so as well here, since Rgb will be free of hardcode to `#000000`. 

**NOTE** I don't introduce new bug here, since it was like that already.

Reply:
Not for all fields, that seems to be pretty much impossible since we have 16 colors. So having 16 different deserialization functions just doesn't make any sense.

Reply:
Hm, so how should I handle deserialization of Rgb then?